src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -783,4 +850,12 @@ ul {
   color: $gray7;
   font-weight: normal;
 }
+
+::v-deep {
```

Comment:
Please add a comment saying what this is for (since it's once-removed from whatever it's styling).

Reply:
Comment added. To override the default font styling of the base address component, form input fields. 
Updated

Reply:
So UI/UX _doesn't_ want the delivery instructions to be italicized here?

(Are you using the latest BaseAddress component? I recently added some classes to it.)

Reply:
That styling still exists here, currently on the latest cc version. Will rm -rf nodeMods and check again. 
There is local styling here to italicize, could be that it is no longer required. Will 👁️

Reply:
i'm not picking up the italics font unless applied manually, locally. Even without any of the overrides in play. Is the component supposed to be applying italic for the delivery instructions?

Reply:
Oops, I forgot that I changed that externally (not in BaseComponent)...

https://github.com/bcgov/business-create-ui/blob/main/src/components/Registration/BusinessAddresses.vue#L370

^^ If UI/UX asks you to do the same.

Reply:
perfect! Updated
fixed what was here.